build.gradle.kts

```diff
@@ -89,6 +89,9 @@ yamlValidator {
 tasks {
 
     test {
+        testLogging {
+            setExceptionFormat("full")
+        }
```

Comment:
What does this do? :blush:

Reply:
The code did not work on linux (and osx I think) and a test faild but only the first line of the exception was shown and not the full stacktrace. You can see it [here](https://travis-ci.org/junit-pioneer/junit-pioneer/jobs/652150699)

> > Task :test
> org.junitpioneer.jupiter.EnvironmentVariableExtensionTests > initializationError FAILED
>     java.lang.NullPointerException at EnvironmentVariableExtensionTests.java:34
> 123 tests completed, 1 failed, 1 skipped

Reply:
Let's keep it, then. :)